parlai/scripts/convo_render.py

```diff
@@ -175,6 +216,13 @@ def display_cli(conversations, alt_speaker, human_speaker):
         type=int,
         default=10,
     )
+    parser.add_argument(
+        "--window-size",
+        "-ws",
+        help="Window Size for screenshot",
+        nargs='+',
+        default=[800, 480],
+    )
```

Comment:
Use the `--width` and `--height` arguments you already have. Just multiply them by 300 (dpi).

Reply:
Tried multiplying them by 300 but that results in a very large conversation and hence everything looks very small. Example:
![b](https://user-images.githubusercontent.com/18496064/66262467-bd6eed00-e7ae-11e9-93f0-20e83097f36e.png)

Using 100 as the mulltiplier seems better:
![b](https://user-images.githubusercontent.com/18496064/66262470-cfe92680-e7ae-11e9-9920-c6afb1063adc.png)

Reply:
Used 100 for now!